src/components/Courses.js

```diff
@@ -260,6 +273,7 @@ class Courses extends React.Component {
     };
 
     const resultsFormatter = {
+      name: r => <TextLink to={this.getRowURL(r.id)} onClick={() => this.setURL(r.id)}>{r.name}</TextLink>,
```

Comment:
Why does this use both `to` _and_ `onClick`?

Reply:
I was following the same logic that was used in the PR referenced in the JIRA ticket,[UIREQ-716](https://github.com/folio-org/ui-requests/pull/849/commits/0afdcb82f0d3ae3e0b4a8bac1d3f1b6296c80297). `to` is used to serve the row URL and `onClick` is used to save the URL in a state variable (which I am adding in this next commit, apologies I forgot to add it in the first commit). Is there a more efficient way I could be doing this?

Reply:
I don't understand why you're saving the URL at all. That bit of state doesn't seem to get used (as is reported by GitHub's built-in linter). Am I missing something?

Reply:
No, you are correct. I was mistaken, the setURL was not necessary after looking back at the functionality. Newest commit should fix all the current issues. Thank you!